app/components/ticket-container.tsx

```diff
@@ -25,17 +23,11 @@ export function TicketContainer({
   ];
   return (
     <div className={classNames.join(" ")}>
-      <div className="ticket-edge-left max-h-full w-4 fill-[var(--theme-color-200)]">
-        <TicketEdgeLeft className="h-full" />
-      </div>
       <div className="ticket-body z-10 max-w-75vw bg-[var(--theme-color-200)] p-2 py-4 sm:max-w-sm md:max-w-lg">
-        <div className="ticket-body-interior flex flex-col space-y-3 rounded border-2 border-solid border-[var(--theme-color-900)] p-2">
```

Comment:
Also removed the border. Looked crummy. I prefer these as index cards - maybe a rework is in order to make them more index-cardy in general.